opm/output/eclipse/EclipseWriter.cpp

```diff
@@ -1205,6 +1223,13 @@ void EclipseWriter::writeInit(const SimulatorTimerInterface &timer)
             EclipseWriterDetails::restrictAndReorderToActiveCells(data, gridToEclipseIdx_.size(), gridToEclipseIdx_.data());
             fortio.writeKeyword("PERMZ", data);
         }
+        if (nnc.hasNNC()) {
+            std::vector<double> tran;
```

Comment:
It's actually implemented. `conversions::from_si( conversion_table, unit, value )`.